packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -776,13 +912,18 @@ export const makeTemplates: AsyncAction<{ sandboxIds: string[] }> = async (
   { sandboxIds: ids }
 ) => {
   const oldSandboxes = state.dashboard.sandboxes;
-  actions.dashboard.deleteSandboxFromState(ids);
+  if (!isRepoPage) {
```

Comment:
I would also make this conditional.

Reply:
Moved the condition to the function itself